### PR TITLE
English Translation for README (caver-js-examples)

### PR DESCRIPTION
**Pull request by jiwon-lieb**
English Translation for README caver-js-examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,74 @@
...
-
-If you are already using a different version of the node(for example, node v14), use the Node Version Manager([NVM](https://github.com/nvm-sh/nvm)) to install and use the version supported by caver-js.
-
-
-## How to use caver-js-boilerplate
-
-1. Download the necessary modules using the comments below.
-
+# caver-js-examples
+**caver-js-examples** provide code samples to help you use the Klaytn SDK ([caver-js](https://github.com/klaytn/caver-js)) easier and faster.
```
Comment (jiwon-lieb) [inline README.md]:
```suggestion
**caver-js-examples** provide code samples to help you use the Klaytn SDK([caver-js](https://github.com/klaytn/caver-js)) easier and faster.
```

File: README.md [hunk 2]
```diff
@@ -1,54 +1,74 @@
...
-If you are already using a different version of the node(for example, node v14), use the Node Version Manager([NVM](https://github.com/nvm-sh/nvm)) to install and use the version supported by caver-js.
-
-
-## How to use caver-js-boilerplate
-
-1. Download the necessary modules using the comments below.
-
+# caver-js-examples
+**caver-js-examples** provide code samples to help you use the Klaytn SDK([caver-js](https://github.com/klaytn/caver-js)) easier and faster.
+Here you can find and test code samples conceived for different scenarios.
```
Comment (jiwon-lieb) [inline README.md]:
```suggestion
**caver-js-examples** provide code samples to help you use the Klaytn SDK([caver-js](https://github.com/klaytn/caver-js)) easier and faster. Here you can find and test code samples conceived for different scenarios.
```

File: README.md [hunk 3]
```diff
@@ -1,54 +1,73 @@
...
+### 1.3 Creating Account
+In order to execute the code samples, you need a test account with a sufficient balance of testnet KLAY.
+Because the code samples also involves sending a transaction to Klaytn, you need at least one test account. If you have updated your account key, and your old test account is rendered unusable, you can create a new account. Create a test account following the instructions below and obtain your 5 testnet KLAY. Depending on the scenarios, you may need multiple test accounts. In that case, just repeat the process to create multiple test accounts.
+1. Create a test account at [Klaytn Wallet Testnet](https://baobab.wallet.klaytn.com/create). 
+    * Have the address and private key of the new account written down, because you will need them for testing the code samples.
+    * Make sure to also save the keystore file, because some scenarios require it.
+2. Obtain 5 testnet KLAY on [Baobab Klaytn Wallet Faucet](https://baobab.wallet.klaytn.com/faucet).
+
+## 2. How to Use
+1. Clone this project and then execute `$ git clone https://github.com/klaytn/caver-js-examples.git` on Terminal.
```
Comment (jiwon-lieb) [inline README.md]:
```suggestion
1. Clone this project and then execute `$ git clone https://github.com/klaytn/caver-js-examples.git` in Terminal.
```

File: README.md [hunk 4]
```diff
@@ -1,54 +1,73 @@
...
+    * Have the address and private key of the new account written down, because you will need them for testing the code samples.
+    * Make sure to also save the keystore file, because some scenarios require it.
+2. Obtain 5 testnet KLAY on [Baobab Klaytn Wallet Faucet](https://baobab.wallet.klaytn.com/faucet).
+
+## 2. How to Use
+1. Clone this project and then execute `$ git clone https://github.com/klaytn/caver-js-examples.git` in Terminal.
+2. To run the samples, you need to set the Credential (AccessKey ID and Secret AccessKey) which can be obtained on [KAS Console](https://console.klaytnapi.com/ko/security/credential) as well as a Klaytn account obtained on [Klaytn Wallet Testnet](https://baobab.wallet.klaytn.com/create). We recommend using the test account just for testing.
+    * Option 1 (recommended): Define the Credential information and your Klaytn account in the **caver-js-examples/.env** file. The variables defined in this file apply for all scenarios. You can find the code for retrieving the variables defined in the **.env** file in the `loadEnv` function in the **index.js** file of each scenario.
```
Comment (jiwon-lieb) [inline README.md]:
```suggestion
    * Option 1 (recommended): Define the Credential information and your Klaytn account in the **caver-js-examples/.env** file. The variables defined in this file apply for all scenarios. You can find the code for retrieving the variables defined in the **.env** file in the `loadEnv` function within **index.js** of each scenario.
```

**Comment by aeharvlee** [comment]
@minishell 
Hi Elena :) Please review this PR.
Thanks in advance.

File: README.md [hunk 5]
```diff
@@ -1,54 +1,73 @@
...
+account
+├── update_account_with_account_key_public
+├── update_account_with_account_key_role_based
+└── update_account_with_account_key_weighted_multisig
 ```
+* Each scenario includes a **index.js** file, and if the scenario requires a specific file, it employs a `resources` directory.
+* This allows the users to execute the samples right away.
 
+### 3.2 Summary
+The caver-js-example project consists of multiple `CA-Layer/시나리오`, and lets you run the code samples with `node CA-Layer/시나리오/index.js` in the project root directory or `node ./index.js` in the scenario's directory.
```
Comment (jiwon-lieb) [inline README.md]:
```suggestion
The caver-js-example project consists of multiple `CA-Layer/Scenario`, and lets you run the code samples with `node CA-Layer/Scenario/index.js` in the project root directory or `node ./index.js` in the scenario's directory.
```

File: README.md [hunk 6]
```diff
@@ -1,54 +1,73 @@
...
-* lts/erbium ([12.19.0](https://nodejs.org/dist/latest-v12.x/))
-
-If you are already using a different version of the node(for example, node v14), use the Node Version Manager([NVM](https://github.com/nvm-sh/nvm)) to install and use the version supported by caver-js.
-
-
-## How to use caver-js-boilerplate
-
-1. Download the necessary modules using the comments below.
-
+# caver-js-examples
```
Comment (jiwon-lieb) [inline README.md:4]:
```suggestion
*Read this in Korean:* [Korean](https://github.com/klaytn/caver-js-examples/blob/master/README.ko.md) 


# caver-js-examples
```

**Review by minishell — APPROVED** [review]

**Pull request merged**